CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ find_package(ament_cmake_vendor_package REQUIRED)
 
 ament_vendor(mimick_vendor
   VCS_URL https://github.com/ros2/Mimick.git
-  VCS_VERSION 4dbd495e9f3d87a83c3201ef9d851e85f7133db7
+  VCS_VERSION d6ed6a70535cf1a2d1f5d9665ae66065ffda7f44
```

Comment:
This hash needs to be updated now, right?  That is, this should be https://github.com/ros2/Mimick/commit/1e138b0e13da99278453dc96af954890d9f48348 ?

Reply:
Done https://github.com/ros2/mimick_vendor/pull/39/commits/136df7282501e4fe628edaf7963da595cb7d1369 and relaunched CI